boards/st/nucleo_c092rc/doc/index.rst

```diff
@@ -0,0 +1,126 @@
+.. zephyr:board:: nucleo_c092rc
+
+Overview
+********
+The STM32 Nucleo-64 development board with STM32C092RC MCU, supports Arduino and ST morpho connectivity.
+
+The STM32 Nucleo board comes with the STM32 comprehensive software HAL library together
+with various packaged software examples.
+
+.. image:: img/st_nucleo_c092rc.webp
+   :align: center
+   :alt: Nucleo C092RC
+
+More information about the board can be found at the `Nucleo C092RC website`_.
+
+Hardware
+********
+Nucleo C092RC provides the following hardware components:
+
+- STM32 microcontroller in 64-pin package featuring 256 Kbytes of Flash memory
+  and 30 Kbytes of SRAM.
+
+- Flexible board power supply:
+
+  - USB VBUS or external source (3.3V, 5V, 7 - 12V)
+  - Current consumption measurement (IDD)
+
+- five LEDs:
+
+  - Two user LEDs (LD1, LD2), power LED (LD3),
+    STLINK LED (LD4), USB power fault LED (LD5)
+
+- Three push-buttons: USER, RESET, BOOT
+
+- On-board ST-LINK/V2-1 debugger/programmer with SWD connector
+
+- Board connectors:
+
+  - Arduino* Uno V3 expansion connector
+  - ST morpho extension pin header
+  - CAN FD interface with on board Transceiver
+
+More information about STM32C092RC can be found here:
+`STM32C0x1 reference manual`_
+
+Supported Features
+==================
+
+.. zephyr:board-supported-hw::
+
+Connections and IOs
+===================
+
+Each of the GPIO pins can be configured by software as output (push-pull or open-drain), as
+input (with or without pull-up or pull-down), or as peripheral alternate function. Most of the
+GPIO pins are shared with digital or analog alternate functions. All GPIOs are high current
+capable except for analog inputs.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- CAN RX/TX/STBY: PD0/PD1/PD2
+- I2C1 SCL/SDA : PB8/PB9 (Arduino I2C)
+- LD1       : PA5
+- LD2       : PC9
+- SPI1 NSS/SCK/MISO/MOSI : PA15/PA5/PA5/PA7 (Arduino SPI)
+- UART_1 TX/RX : PB6/PB7 (Arduino Serial)
+- UART_2 TX/RX : PA2/PA3 (ST-Link Virtual Port Com)
+- USER_PB : PC13
+
+
+For more details please refer to `STM32 Nucleo-64 board User Manual`_.
+
+Programming and Debugging
+*************************
+
+.. zephyr:board-supported-runners::
+
+Nucleo C092RE board includes an ST-LINK/V2-1 embedded debug tool interface.
```

Comment:
```suggestion
Nucleo C092RC board includes an ST-LINK/V2-1 embedded debug tool interface.
```

Please check for similar typos you might have elsewhere

Reply:
Thanks for spotting the typo. Should be fixed now.